example/ssd/data/demo/download_demo_images.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python3
```

Comment:
Do we have a standard for headers?  It'd be great if we could follow a convention for what to put here.

Reply:
Would be great. As far as I know this is the recommended best practice.